modules/configuring-default-route-timeouts.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// * knative-serving/config-applications/configuring-revision-timeouts.adoc
+:_content-type: PROCEDURE
+[id="configuring-default-route-timeouts_{context}"]
+= Configuring the default route timeouts per revision
+
+By configuring the route timeouts per revision, you can fine-tune timeouts for workloads with unique requirements, such as AI or data processing applications, without impacting the global timeout settings for other services.
+You can configure the timeouts of a specific revision by updating your `KnativeServing` custom resource (CR), the Service definition, and using the `serving.knative.openshift.io/setRouteTimeout` annotation to adjust the OCP route timeout.
+
+.Procedure
+
+. Set the `max-revision-timeout` annotation in your `KnativeServing` CR as you require:
++
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1beta1
+kind: KnativeServing
+metadata:
+  name: knative-serving
+spec:
+  config:
+    defaults:
+      max-revision-timeout-seconds: "900"
+----
++
+You can verify the activator's termination grace period by running the following command:
++
+[source,terminal]
+----
+$ oc get deployment activator -n knative-serving -o jsonpath="{.spec.template.spec.terminationGracePeriodSeconds}"
+900
+----
+
+. Optional: Adjust the load balancer timeout if necessary for your cloud provider:
++
+.Example of load balancer timeout adjustment for AWS Classic LB 
+[source,terminal]
+----
+$ oc -n openshift-ingress-operator patch ingresscontroller/default --type=merge --patch='{"spec":{"endpointPublishingStrategy": {"type":"LoadBalancerService", "loadBalancer": {"scope":"External", "providerParameters":{"type":"AWS", "aws": {"type":"Classic", "classicLoadBalancer": {"connectionIdleTimeout":"20m"}}}}}}}'
+----
+
+. Set the timeout for your specific service:
++
+[source,yaml]
+----
+apiVersion: serving.knative.dev/v1f
+kind: Service
+metadata:
+  name: <your_service_name>
+  annotations:
+    serving.knative.openshift.io/setRouteTimeout: "800" <1>
+spec:
+  template:
+    metadata:
+      annotations:
+#...
+    spec:
+      timeoutSeconds: 800 <2>
+      responseStartTimeoutSeconds: 800 <3>
+----
+<1> This annotation sets the timeout for the OCP route. You can fine-tune this for each service instead of setting a global maximum.
```

Comment:
```suggestion
<1> This annotation sets the timeout for the OpenShift route. You can fine-tune this for each service instead of setting a global maximum.
```

Same comment about removing "OCP"